src/logic/operate.js

```diff
@@ -0,0 +1,34 @@
+import Big from 'big-js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const numOne = Big(numberOne);
+  const numTwo = Big(numberTwo);
+
+  let total;
+
+  if (operation === '+') {
+    total = numOne.plus(numTwo).toString();
+  } else if (operation === '-') {
+    total = numOne.minus(numTwo).toString();
+  } else if (operation === '×') {
+    total = numOne.times(numTwo).toString();
+  } else if (operation === '÷') {
+    if (numTwo.eq(0)) {
+      total = 'Division by zero is undefined';
+    } else {
+      total = numOne.div(numTwo).toString();
+    }
+  } else if (operation === '%') {
+    if (!numTwo) {
+      total = numOne.times(0.01).toString();
+    } else {
+      total = (numOne.times(numTwo)).times(0.01).toString();
+    }
+  } else {
+    total = '0';
+  }
+
+  return total;
```

Comment:
Instead of repeating `toString()` in all cases, why not use it only once, when we about to return the `total` value above? 🤔

Reply:
Updated as you suggested,